### PR TITLE
Added Basic pricing for smaller merchants

### PR DESCRIPTION
**Pull request by octaveauger**
@adeclosset let me know what you think. We need to review our deals to make sure we have no one on Pro pricing that is collecting < 500 payments a month. I still think 500 is quite high.

@jamesshedden are you happy with the layout? I've made the Pro pricing blue as we want to attract attention to that one compared to the UK version.

@hirokitakeuchi @greysteil fyi

image

### CODE REVIEW

**Comment by jamesshedden** [comment]
@octaveauger Hey, nice idea about the blue - but we probably don't want to change that for the moment, as generally throughout splash pages we're using the blue vs grey to differentiate between GC & GC Pro

File: app/pages/fr/tarifs/index.html [hunk 1]
```diff
@@ -19,35 +19,68 @@
...
-          <h1 class="u-text-heading-light u-text-center u-color-invert u-text-l u-padding-Bs">GoCardless</h1>
-          <div class="u-background-dark-gray-darken u-padding-Vxl">
-            <h2 class="u-text-heading-light u-text-center u-color-invert u-text-xl">0,01 — 0,75 €</h2>
-            <p class="u-text-heading-light u-text-center u-color-invert u-text-s u-margin-Txxs">
-              Par transaction
-            </p>
+      <div class="grid pricing-options u-center u-padding-Bxl">
+        <div class="grid__cell u-size-1of2 u-padding-Vxl u-padding-Rxs">
+
+          <div class="grid__cell u-padding-Vxl u-padding-Rxs">
```
Comment (jamesshedden) [inline app/pages/fr/tarifs/index.html]: Change 

`<div class="grid__cell u-padding-Vxl u-padding-Rxs">`

to

`<div class="u-padding-Rxs">`

File: app/pages/fr/tarifs/index.html [hunk 2]
```diff
@@ -19,35 +19,68 @@
...
-            <li class="pricing-options__list-button">
-              <a href="/fr/contactez-nous?s=pricing" class="btn btn--hollow u-size-full">Contactez-nous</a>
-            </li>
-          </ul>
+
         </div>
 
+        <div class="grid__cell u-size-1of2 u-padding-Vxl u-padding-Rxs">
+
+          <div class="grid__cell u-padding-Vxl u-padding-Rxs">
```
Comment (jamesshedden) [inline app/pages/fr/tarifs/index.html]: Change 

`<div class="grid__cell u-padding-Vxl u-padding-Rxs">`

to

`<div class="u-padding-Lxs">`

File: app/pages/fr/tarifs/index.html [hunk 3]
```diff
@@ -19,35 +19,68 @@
...
+            </div>
+            <ul class="pricing-options__list">
+              <li class="pricing-options__list-item u-color-heading">
+                <b>Pour les entreprises qui souhaitent un contrôle total</b>
+              </li>
+              <li class="pricing-options__list-item">
+                <b>+ 250 € par mois</b>, pas de frais cachés
+              </li>
+              <li class="pricing-options__list-item">
+                Dégressif à partir de <b>1.000 paiements</b> par mois
```
Comment (jamesshedden) [inline app/pages/fr/tarifs/index.html]: Change

`Dégressif à partir de <b>1.000 paiements</b> par mois`

to

`Dégressif à partir de <br><b>1.000 paiements</b> par mois`

**Comment by jamesshedden** [comment]
@octaveauger Few comments just to tweak the layout a bit, nice work! :fr: :+1:

**Comment by adeclosset** [comment]
@octaveauger I've made a couple of changes, can you check? Think we need to do a wider re-work of product to split it in Basic / Pro like in the UK. This will do for the moment.

@jamesshedden - FYI made the changes you mentioned!

**Comment by octaveauger** [comment]
All changes done according to @jamesshedden recommendations and top pricing decreased to €0.60 - merging now.

**Pull request merged**